cli/command/formatter/stats.go

```diff
@@ -25,7 +25,9 @@ const (
 
 // StatsEntry represents represents the statistics data collected from a container
 type StatsEntry struct {
+	Container        string
 	Name             string
+	ID               string
```

Comment:
Check my understanding.
+ Container has the user input name.
+ Name has the full name, and used to have the user input name.
+ ID has the full ID.